Technology/Artificial Intelligence/Maths/prerequisite_for_AI/beginner/README.md

```diff
@@ -0,0 +1,18 @@
+
+## Prerequisite for Machine Learning
+
+  
+
+> With our rich expertise we created prodigious stage wise self-starter content just for you in :
+> 
+> 
+> 
+> #### 1. Linear Algebra

```

Comment:
Add hyperlink for the folders you created.

Also. Instead of writing with _ write like Prerequisite for AI or Linear Algebra